src/Notify/Models/Responses/LetterNotificationResponse.cs

```diff
@@ -1,25 +1,31 @@
-﻿using System;
-using Newtonsoft.Json;
-
-namespace Notify.Models.Responses
+﻿namespace Notify.Models.Responses
 {
     public class LetterNotificationResponse : NotificationResponse
     {
         public LetterResponseContent content;
 
-        public bool IsEqualTo(LetterNotificationResponse response)
+        public override bool Equals(object response)
+        {
+            if (!(response is LetterNotificationResponse resp))
```

Comment:
Wouldn't the below do the same?

return 
    response is LetterNotificationResponse resp && 
    content.body == resp.content.body &&
    content.subject == resp.content.subject &&
    base.Equals(resp);